pipeline/00-ingest.R

```diff
@@ -395,6 +410,22 @@ assessment_data_clean <- assessment_data_w_hie %>%
     type = "short",
     as_factor = FALSE
   ) %>%
+  # Same array cleanup as training data
```

Comment:
When the changes were implemented after the code on line 441, it would error out, thus it's moved slightly up from it's original location.